desktopVersion.css

```diff
@@ -0,0 +1,502 @@
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style: none;
+  }
+
+  body {
+    background-color: #f1f1f1f1;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+    background-color: white;
+    align-items: center;
+    padding: 1rem;
+  }
+
+  .logo {
+    font-family: "Poppins", sans-serif;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .navbar__ul {
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+    height: 20px;
+  }
+
+  .navbar__ul li {
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #344563;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .description {
+    background-color: white;
+    background-image: url(img/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position-y: 9px;
+    border-bottom-left-radius: 6rem;
+    padding-bottom: 4rem;
+    height: 700px;
+    margin-top: -0.1rem;
+  }
+
+  .header {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    padding: 100px 150px 2px 200px;
+    font-size: 48px;
+    text-align: left;
+    line-height: 60px;
+  }
+
+  .parg {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    color: #344563;
+    padding: 15px 100px 2px 200px;
+    width: 90%;
+
+    /* padding: 16px 162px 0px 162px; */
+    font-size: 20px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .let__go {
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    padding: 1.2rem 100px 0 200px;
+    text-align: left;
+    color: #396df2;
+  }
+
+  .social__media {
+    display: flex;
+    padding: 12px 100px 0 200px;
+  }
+
+  .social__media__link {
+    margin-right: 1rem;
+    height: 20px;
+    width: 20px;
+  }
+
+  .social__media__img {
+    width: 13px;
+    color: #505f79;
+  }
+
+  /* ====================================POST============================================== */
+
+  .post {
+    margin-top: 3rem;
+    background-color: #f1f1f1;
+    padding: 0 70px;
+  }
+
+  .post__container {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Great job implementing required changes from the previous reviewer. However, there're few things to take care before you move on.
- [ ] Kindly ensure that you produce a close replica of the figma design at every section. I noticed that the width and the height of the project image is too much and as a result it affects how the project info should align beside the image. Kindly check the two screenshots and walk your design alongside the figma design. Pay attention to those areas I marked with red.

### Your Design
![your-project-card](https://github.com/sibomana123/MV__Portfolio/assets/26597432/770ec798-b54c-4b3e-94bf-e75807e060ea)

### Figma Design
![temp-project=card](https://github.com/sibomana123/MV__Portfolio/assets/26597432/230fce07-0bcc-48cf-b713-81850643569c)